accounting/invoice/choices.py

```diff
@@ -74,6 +74,7 @@ class InvoiceNumberingScheme(DjangoChoices):
     year_month_number = ChoiceItem('%Y-%m-{number}', _('yyyy-mm-{number} (2018-01-42, for the 42nd invoice)'))
     opencraft_year_month = ChoiceItem('OC-%Y-%m', _('OC-yyyy-mm (OC-2018-01)'))
     opencraft_number = ChoiceItem('OC-{number}', _('OC-{number} (OC-42, for the 42nd invoice)'))
+    year_month_one = ChoiceItem('%Y-%m-01', _('yyyy-mm-01 (2018-01-01, as just one is expected)'))
```

Comment:
@viadanna Could you move this option up so it sits below `year_month_number`? It's more closely related to that format than the `opencraft_*` numbering schemes.